package proto_ff;

// 服务器内部的消息id，是个统一的
//
enum Proto_SvrLogicMsgID
{
  ///////////////////////////login/////////////////////////////////////
  NF_WTL_PLAYER_LOGIN_REQ = 130;      //world to logic, login msg
  NF_LTW_PLAYER_LOGIN_RSP = 131;      //logic to world, login rsp msg
  NF_PTW_PLAYER_LOGIN_REQ = 132;      //proxy to world, login msg
  NF_WTP_PLAYER_LOGIN_RSP = 133;      //world to proxy, login rsp msg
  NF_LTS_PLAYER_LOGIN_REQ = 134;      //logic to sns, login msg
  NF_STL_PLAYER_LOGIN_RSP = 135;     //sns to logic, login rsp msg
  NF_LTS_PLAYER_LOGOUT_REQ = 136;      //logic to sns, logout msg
  NF_STL_PLAYER_LOGOUT_RSP = 137;     //sns to logic, logout msg
  NF_LTW_PLAYER_LOGOUT_REQ = 138;     //logic to world, logout msg
  NF_WTL_PLAYER_LOGOUT_NOTIFY = 139;  //world to logic, notify msg
}
